### PR TITLE
Add more tile intrinsic functions

### PR DESCRIPTION
**Pull request by vpipkt**
* round
* log including base 10, base 2, and minus 1
* exp including base 10, base 2, and plus 1

### CODE REVIEW

File: core/src/main/scala/astraea/spark/rasterframes/expressions/localops/Round.scala [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+       ..."""
+)
+case class Round(child: Expression) extends UnaryLocalRasterOp
+  with NullToValue with CodegenFallback {
+  override def nodeName: String = "round"
+  override def na: Any = null
+  override protected def op(child: Tile): Tile = child.localRound()
+}
+object Round{
+//  import astraea.spark.rasterframes.encoders.StandardEncoders.PrimitiveEncoders.doubleEnc
```
Comment (vpipkt) [inline core/src/main/scala/astraea/spark/rasterframes/expressions/localops/Round.scala]: remove

**Comment by vpipkt** [comment]
Should I have added tests to `PyRasterFramesTests.py`?

**Review by metasim — CHANGES_REQUESTED** [review]
Can you add update the reference section of the manual you wrote with your additions. We need to start keeping all those bits in sync. Of course I'm tempted to have that page automatically generated in code, but I know that's not what I need to be working on now.... ;-)

**Comment by metasim** [comment]
@vpipkt The Python tests are really weak right now... I'd like to get them to the place where each function is at least called from Python, as it will scare up most problems. Not putting that all on you, but small steps are big steps.

**Comment by vpipkt** [comment]
@metasim I'll put up a PR for the docs changes and you can merge at the same time. I'll probably put up a little work on the python tests as it will help me with both efforts.

**Comment by vpipkt** [comment]
See doc update here https://github.com/s22s/earthai-documentation/pull/23

**Review by metasim — APPROVED** [review]

**Pull request merged**